src/Take.Blip.Builder/IInputMessageHandler.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Lime.Protocol;
+
+namespace Take.Blip.Builder
+{
+    public interface IInputMessageHandler
+    {
+        (bool, Message) ValidateMessage(Message message);
```

Comment:
Using tuple on public APIs isn't recommended - see the last paragraph [here](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples#use-cases-of-tuples)

But if you keep it, I suggest to add field names to it,  to better communicate their semantic. For instance
(bool MessageHasChanged, Message NewMessage)

Reply:
We have decided to keep using the tuple to avoid making significant changes to the existing logic. We have applied the previous suggestion by adding names to the tuple fields, which improves clarity and makes the purpose of each value more evident.